ui/analyse/src/ground.ts

```diff
@@ -10,7 +10,6 @@ export const render = (ctrl: AnalyseCtrl): VNode =>
   h('div.cg-wrap.cgv' + ctrl.cgVersion.js, {
     hook: {
       insert: vnode => ctrl.setChessground(makeChessground(vnode.elm as HTMLElement, makeConfig(ctrl))),
-      destroy: _ => ctrl.chessground.destroy(),
```

Comment:
how did you determine that this could be removed?

Reply:
Nvui relies on chessground to keep track of board state. That line destroys the initialized chessground state whenever the main analysis board is removed from the DOM tree (which is the order of operations on every analysis/study page load)